### PR TITLE
#1 only import the cssParse function we need

### PR DESCRIPTION
**Pull request by IanGrainger**
**What**:

We can't import the CSS library in the browser (which is how Karma runs the tests) - because of an error `Module not found: Error: Can't resolve 'fs'`. But as suggested by a developer on that project, we can just import the function we need from the library.

**Why**:

Because attempting to run tests with Karma results in an error: `Module not found: Error: Can't resolve 'fs'`.

**How**:

Import the cssParse function we need rather than the whole css library.

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [ ] Updated Type Definitions
- [x] Ready to be merged

### CODE REVIEW

**Comment by IanGrainger** [comment]
Oh dear. Looks like a caching problem in your build script? :(

**Review by ogonkov — APPROVED** [review]
Looking forward to see it merged

File: src/toHaveStyle.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import { parse } from 'css';
 import { checkHtmlElement, getTag, InvalidCSSError } from './utils';
 import { printSecSuccess, printSuccess, printSecError, printError } from './printers';
+import cssParse from 'css/lib/parse';
```
Comment (ogonkov) [inline src/toHaveStyle.js:3]: I guess that should be on the first line in this file, since it is module dependency, then own dependencies goes

**Review by ogonkov — CHANGES_REQUESTED** [review]

**Comment by brrianalexis** [comment]
> Oh dear. Looks like a caching problem in your build script? :(

The Office - MIchael Scott -I Don't Know What The Fuck That Was
Maybe running actions on windows wasn't such a great idea 😅

I'll try this out on the weekend @IanGrainger! Let's hope for the best.

Thank you for your support and your efforts.

**Comment by IanGrainger** [comment]
I've pushed changing the import order. Sorry about that! 😅 

Hey, how'd the new job go? Did you get it?? 😄

**Review by ogonkov — APPROVED** [review]

**Review by brrianalexis — APPROVED** [review]

**Comment by brrianalexis** [comment]
I did get a new job, thanks for asking! 😄

Thank you for putting so much work into this.

**Comment by brrianalexis** [comment]
@all-contributors add @IanGrainger for code

**Pull request merged**